compiler/compiler.py

```diff
@@ -376,6 +385,11 @@ def _run(source: SourcePackage, output_format: Format = Format.PDF,
         (id_for_decryption is not None, f'-d {id_for_decryption}'),
         (tex_tree_timestamp is not None, f'-U {tex_tree_timestamp}')
     ]
+    if stamp_label:
+        options.append((True, f'-l "{stamp_label}"'))
+    if stamp_link:
+        options.append((True, f'-L "{stamp_link}"'))
```

Comment:
Any reason not to add these in the same way as the options above? E.g. 

```python
options = [
    ...
    (stamp_label is not None, f'-l "{stamp_label}"'),
    (stamp_link is not None, f'-L "{stamp_link}"')
]
```

Reply:
Move into options list.